src/components/Header/AccountMenu/account-menu.module.css

```diff
@@ -0,0 +1,42 @@
+.container {
+  display: flex;
+  align-items: center;
+
+  gap: 10px;
+
+  height: 46px;
+
+  padding: 8px;
+
+  background: none;
+
+  border: 1px solid #0F121D;
```

Comment:
There is also a possibility of leaving all these hexadecimal values in variables

if it is not necessary to separate it into a file, it makes sense to leave it at the top of the file